src/basilisp/lang/map.py

```diff
@@ -107,6 +125,78 @@ def to_persistent(self) -> "PersistentMap[K, V]":
         return PersistentMap(self._inner.finish())
 
 
+def map_lrepr(  # pylint: disable=too-many-locals
+    entries: Callable[[], Iterable[Tuple[Any, Any]]],
+    start: str,
+    end: str,
+    meta=None,
```

Comment:
```suggestion
    meta: Optional[IPersistentMap] = None,
```

I think this was too hard to type correctly in the previous location, but now it should be fine.

Reply:
updated, thanks